packages/teleport/src/Databases/ConnectDialog/ConnectDialog.tsx

```diff
@@ -68,7 +68,7 @@ export default function ConnectDialog({
             Step 2
           </Text>
           {' - Retrieve credentials for the database'}
-          <TextSelectCopy mt="2" text={`tsh db login ${dbName}`} />
+          <TextSelectCopy mt="2" text={`tsh db login [--db-user=<user>] [--db-name=<name>] ${dbName}`} />
```

Comment:
`[--db-user=<user>]`, does the square bracket mean optional?

Reply:
Yeah

Reply:
shoot, i feel like we have to go back and change all square brackets to angle brackets to keep the meaning consistent, b/c we currently use square brackets to mean `user need to fill this out`, maybe as a refactor for later? @alex-kovoy